k8s.gcr.io/images/k8s-staging-infra-tools/images.yaml

```diff
@@ -3,6 +3,7 @@
     "sha256:2b44fee8ac3db2e4de7939a1a9855e425dee76d6d9b7ac333b2077a16ae5bd66": ["v20220331-v0.0.1-2-g9b73f75"]
     "sha256:9b334d3df84e1719adcb09472f01ab916db1a01e2e773cb2ab81352f6204d086": ["v20220331-v0.0.1-4-gc3b27f3"]
     "sha256:4c723dd3ebedcf00df36f0b20b05714fac7f6e032f85fe221422996b2406c2bf": ["v20220816-v0.0.1-120-g8293873"]
+    "sha256:df959a5384c570000dfe396d38b3c0102136d0753c71c38507a1b3603f749dc5": ["v20221005-v0.0.2-28-g52c8633"]
```

Comment:
should we do v0.0.3 then?

Reply:
i mean here, perhaps the tag we promote should match /shrug

Reply:
Let's do that for `v0.1.0`.